skins/laika/src/store/address/mutations.ts

```diff
@@ -19,7 +19,7 @@ import { REQUIRED_FIELDS } from '@/store/address/constants';
 export const mutations: MutationTree<AddressState> = {
 	[ VALIDATE_INPUT ]( state: AddressState, field: InputField ) {
 		if ( field.value === '' && field.optionalField ) {
-			state.validity[ field.name ] = Validity.INCOMPLETE;
+			state.validity[ field.name ] = Validity.VALID;
```

Comment:
hmmm, this does solve the problem but now I'm thinking our logic is confusing.
There is a constant for required fields used in store modules and there's the property `optionaldField` used in vue components. Not cool.
Anyway that's a topic for another PR.